### ISSUE TITLE
Add a ZSTD_defaultCLevel() function

### ISSUE THREAD

**Issue by ghost** [facebook/zstd#2531]
The stable API has these two functions:
```c
int  ZSTD_minCLevel(void);  /*!< minimum negative compression level allowed */
int  ZSTD_maxCLevel(void);  /*!< maximum compression level available */
```
Does it need a `ZSTD_defaultCLevel()` function that returns the value of `ZSTD_CLEVEL_DEFAULT`?

**Comment by you-me227** [comment]
> `
Duplicate of #`

**Comment by ghost** [comment]
It can be used when dynamically linking to zstd.

This request is not strong, maybe very few people modify the value of `ZSTD_CLEVEL_DEFAULT`.

**Comment by terrelln** [comment]
I think that very few people modify it, but since it is modifiable, and there is no other way to detect the default level, I wouldn't be against adding it.

**Comment by senhuang42** [comment]
Included in #2555
